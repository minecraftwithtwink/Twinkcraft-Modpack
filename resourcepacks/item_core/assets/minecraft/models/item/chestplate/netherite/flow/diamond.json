{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/chestplate/base/netherite/flow",
        "layer1": "minecraft:trims/items/chestplate_trim/flow_diamond"
    }
}