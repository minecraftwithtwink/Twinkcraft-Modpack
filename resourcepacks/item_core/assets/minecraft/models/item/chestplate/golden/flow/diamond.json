{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/chestplate/base/golden/flow",
        "layer1": "minecraft:trims/items/chestplate_trim/flow_diamond"
    }
}